### ISSUE TITLE
Build fail when using rust nightly 

### ISSUE THREAD

**Issue by bjorn3** [kevinmehall/rust-peg#131]
It looks like this line is broken: https://github.com/kevinmehall/rust-peg/blob/master/src/rustast.rs#L21

Rust version info:
```
$ rustc --version
rustc 1.10.0-nightly (645dd013a 2016-04-24)
$ cargo --version
cargo 0.11.0-nightly (191f65f 2016-04-23)
```

**Comment by kevinmehall** [comment]
Fixed in 4b0b08d5f / 0.3.11

**Comment by bjorn3** [comment]
Thanks
